aibolit/metrics/external_methods_called/external_methods_called.py

```diff
@@ -17,8 +15,8 @@ class ExternalMethodsCalled:
     def __init__(self):
         pass
 
-    def value(self, filepath: str | os.PathLike):
-        return ExternalMethodsCalledCount(AST.build_from_javalang(build_ast(filepath))).total()
+    def value(self, ast: AST) -> int:
```

Comment:
@KachanovYev the problem is that the tests on `ExternalMethodsCalled` still pass `filepath` into method `value`. The tests need to be updated accordingly, so that the conversion from the filepath to `AST` is done there.